lib/flop_phoenix.ex

```diff
@@ -388,6 +414,8 @@ defmodule Flop.Phoenix do
     `<tbody>`. Default: `#{inspect(@default_table_opts[:tbody_tr_attrs])}`.
   - `:tbody_td_attrs`: Attributes to added to each `<td>` tag within the
     `<tbody>`. Default: `#{inspect(@default_table_opts[:tbody_td_attrs])}`.
+  - `:event_name`: If set, tells `Flop.Phoenix` to use event based sorting.
+    Default: nil
```

Comment:
The defaults are set as module attributes and dynamically inserted into the doc. We should do that here as well.